cmd/controller/controller.go

```diff
@@ -288,7 +288,11 @@ func (c *Controller) unseal(key string) (unsealErr error) {
 
 func (c *Controller) updateSealedSecretStatus(ssecret *ssv1alpha1.SealedSecret, unsealError error) error {
 	ssecret = ssecret.DeepCopy()
-	st := &ssecret.Status
+	st := ssecret.Status
+	if st == nil {
+		st = &ssv1alpha1.SealedSecretStatus{}
+	}
+	ssecret.Status = st
```

Comment:
Maybe ...

```
if ssecret.Status == nil {
    ssecret.Status = &ssv1alpha1.SealedSecretStatus{}
}
st := ssecret.Status
```

Could even drop `st` now the secret's `Status` is a pointer value, although it does reduce a bit of line noise.

Reply:
yeah it was ugly